src/operator/gitea.ts

```diff
@@ -130,10 +130,29 @@ const secretsAndConfigmapsCallback = async (e: any) => {
   }
 }
 
+const createSetGiteaOIDCConfig = (() => {
+  let intervalId: any = null
+  return function runSetGiteaOIDCConfig() {
+    if (intervalId === null) {
+      intervalId = setInterval(() => {
+        setGiteaOIDCConfig()
+          .catch((error) => {
+            console.error('Error occurred during setGiteaOIDCConfig execution:', error)
+          })
+          .finally(() => {
+            intervalId = null
+          })
+      }, 30 * 1000)
```

Comment:
the interval should be configurable via ENV

Reply:
ENV added for the interval